app/services/log_event_searcher.rb

```diff
@@ -5,8 +5,8 @@ class LogEventSearcher
   ALLOWED_EVENTS = ["account.create", "account.update",
                     "account_user.create", "account_user.delete",
                     "user.create", "user.suspended", "user.unsuspended",
-                    "account.suspended", "account.unsuspended",
-                    "journal.create","statement.create",
+                    "user.internal_changed", "account.suspended",
```

Comment:
This is kind of picky, but can we leave the two account events on the same line since they're related to each other?